Symfony/CS/Tests/FixerFactoryTest.php

```diff
@@ -279,4 +279,24 @@ function ($name) {
 
         return $cases;
     }
+
+    public function testHasRule()
```

Comment:
Don't understand, I did this based on other tests:

```php
public function testHasRule()
{
    $factory = new FixerFactory();
    $factory->registerBuiltInFixers();

    $f1 = $this->getMock('Symfony\CS\FixerInterface')
        ->expects($this->any())->method('getName')->willReturn('f1');
    $f2 = $this->getMock('Symfony\CS\FixerInterface')
        ->expects($this->any())->method('getName')->willReturn('f2');
    $f3 = $this->getMock('Symfony\CS\FixerInterface')
        ->expects($this->any())->method('getName')->willReturn('f3');
    $factory->registerFixer($f1);
    $factory->registerCustomFixers(array($f2, $f3));

    $this->assertTrue($factory->hasRule('f1'));
    $this->assertTrue($factory->hasRule('f2'));
    $this->assertTrue($factory->hasRule('f3'));
    $this->assertFalse($factory->hasRule('dummy'));
}
```

But I got:

```
1) Symfony\CS\Tests\FixerFactoryTest::testHasRule
Argument 1 passed to Symfony\CS\FixerFactory::registerFixer() must implement interface Symfony\CS\FixerInterface, instance of PHPUnit_Framework_MockObject_Builder_InvocationMocker given, called in /home/sullivan/projects/fork/PHP-CS-Fixer/Symfony/CS/Tests/FixerFactoryTest.php on line 294 and defined

/home/sullivan/projects/fork/PHP-CS-Fixer/Symfony/CS/FixerFactory.php:133
/home/sullivan/projects/fork/PHP-CS-Fixer/Symfony/CS/Tests/FixerFactoryTest.php:294
```

And it is a `FixerInterface` mock... any idea?